package.json

```diff
@@ -28,10 +28,13 @@
   "dependencies": {
     "asset-pipe-common": "^1.0.0-beta.6",
     "browserify": "^15.2.0",
+    "envify": "^4.1.0",
```

Comment:
https://github.com/zertosh/loose-envify moar speed? not sure if needed

Reply:
Without the uglifyify step, is there much point in using envify at all? I guess it can't hurt.

Reply:
Good point! I'm in the "I guess it can't hurt." camp